splinterd/src/node/running/mod.rs

```diff
@@ -85,11 +85,17 @@ impl Node {
         self.network_subsystem.network_endpoints()
     }
 
-    pub fn admin_service_client(self: &Node) -> Box<dyn AdminServiceClient> {
-        Box::new(ReqwestAdminServiceClient::new(
-            format!("http://localhost:{}", self.rest_api_port),
-            "foo".to_string(),
-        ))
+    pub fn admin_service_client(self: &Node, auth: Option<String>) -> Box<dyn AdminServiceClient> {
```

Comment:
This method feels like the wrong place to inject the auth string (the size of the patch as evidence).